source/_posts/Proof-of-concept-met-azure-batch.md

```diff
@@ -0,0 +1,447 @@
+---
+title: Proof of concept met Azure Batch
+date: 2019-05-01 19:54:37
+tags:
+---
+
+Case omschrijving 
+---
+
+Net als in mijn vorige blog post (https://cloudrepublic.github.io/2019/03/28/Proof-of-concept-met-durable-functions/) heb ik een Proof of Concept gemaakt om grote hoeveelheden XML bestanden te transformeren het gaat dan 3000 bestanden per keer met een totale grote van 18 gigabyte. Voor dit artikel kan ik wegens privacy niet de echte data gebruiken en heb ik een dataset gebruikt van https://www.kaggle.com/datasets het gaat hier om een dataset met landen en de wijnen welke uit het desbetreffende land komen. 
+
+Het doel is om de wijnen uit de XML dump te halen en deze om te zetten naar een JSON formaat en deze bestanden te uploaden in een blob container. We krijgen dus per wijn een JSON bestand in een blob container. Dit alles moet gebeuren op basis Azure Batch met een Azure Function als orchestrator
+
+Wat is de opzet van de POC
+---
+We gaan de xml bestanden transformeren doormiddel van een Azure Batch component en we starten en beheren de Azure Batch doormiddel van een Azure Function.
+De Azure Function zal de xml bestanden toevoegen aan een Job in Azure Batch doormiddel van een event gridt rigger (https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-event-grid) en via een httptrigger is de voortgang te zien van het batch proces. 
+
+<img src="/images/azure-batch-overview.png" />
+
+Wat is Azure Batch
+---
+Azure Batch is plat gezegd eigenlijk een beheer tool om virtuele machines mee te beheren. Elk van deze machines kan een taak oppakken en dit als input gebruiken voor een commandline applicatie en het resultaat uploaden in een blob container. 
+
+Voor de complete omschrijving wat je met Azure Batch kunt doen verwijs ik je graag door naar de Microsoft site https://azure.microsoft.com/nl-nl/services/batch/
+
+Hoe werkt dit nu eigenlijk allemaal
+---
+
+Azure Batch bestaat uit een aantal componenten.
+
+* We hebben een Task, dit is en opdracht welke uitgevoerd dient te worden op een node 
```

Comment:
Een Task is een opdracht welke uitgevoerd dient te worden op een node.
Haal dat "we hebben een" weg